src/main/java/uk/gov/ons/ctp/response/action/scheduled/ReportScheduler.java

```diff
@@ -35,21 +35,23 @@
   @PostConstruct
   public void init() {
     reportDistributedInstanceManager.incrementInstanceCount(DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT);
-    log.info(
-        "{} {} instance/s running",
-        reportDistributedInstanceManager.getInstanceCount(DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT),
-        DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT);
+    log.with(
+            DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT,
```

Comment:
is this key snake_case?

Reply:
`DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT = "reportscheduler"` not quite snake case but this is the actual name given to the instance

Reply:
changed to `.with("instance_key", DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT)` and `.with("instance_count",...`